clean.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
It looks like most of these are in the `.gitignore` so I don't think this script is necessary to add.